Code/educopter_frontend/lib/create_mission/view/create_mission.dart

```diff
@@ -34,176 +34,231 @@ class _MissionCreateScreenState extends State<MissionCreateScreen> {
+                              labelText: 'Maximum aantal inwoners',
+                              saveValue:
+                                  missionCriteria.setMaxPopulationFilter),
+                        ),
+                        Flexible(
+                          child: CheckboxListTile(
+                              controlAffinity: ListTileControlAffinity.leading,
+                              title: Text('Statische route?'),
+                              value: missionCriteria.fixedRoute,
+                              onChanged: (bool? value) {
+                                setState(() {
+                                  missionCriteria.fixedRoute = value!;
+                                });
+                              }),
+                        ),
+                        Flexible(
+                          child: CheckboxListTile(
+                              controlAffinity: ListTileControlAffinity.leading,
+                              title: Text('Alleen hoofdsteden'),
+                              value: missionCriteria.countryCapitalFilter,
+                              onChanged: (bool? value) {
+                                setState(() {
+                                  missionCriteria.countryCapitalFilter = value!;
+                                });
+                              }),
+                        ),
+                        Flexible(
+                          child: CheckboxListTile(
+                              controlAffinity: ListTileControlAffinity.leading,
+                              title: Text('Alleen provincie-hoofdsteden'),
+                              value: missionCriteria.stateCapitalFilter,
+                              onChanged: (bool? value) {
+                                setState(() {
+                                  missionCriteria.stateCapitalFilter = value!;
+                                });
+                              }),
+                        ),
+                        ConstrainedBox(
+                          constraints: BoxConstraints(maxWidth: 150),
+                          child: CustomFormField(
+                              numOnly: true,
+                              labelText: 'Aantal Steden',
+                              saveValue: missionCriteria.setDestinationCount),
+                        ),
+                        SizedBox(width: 10),
+                      ],
                     ),
-                    ElevatedButton(
-                      onPressed: () {
-                        final form = formKey.currentState!;
-                        if (form.validate()) {
-                          form.save();
-                          print('Formulier gevalideerd ik save');
-                        }
-                      },
-                      child: Text('Filter steden'),
+                    Row(
+                      children: [
+                        // ElevatedButton(
```

Comment:
Comment weghalen als  de code niet meer relevant is

Reply:
Deze is nog wel relevant. Ik ben er nog niet uit hoe we om moeten gaan met updates van waarden in de velden min/max inwoners. Als deze onchanged de lijst updaten dan is dat zeer inefficient: stel je wilt 10000 typen dan update hij op 1, 10, 100, etc en dan stelt hij telkens een nieuwe totaalfilter samen en filtert hij de complete lijst. Als ik hier geen fix voor weet te vinden dan moet deze knop gebruikt worden om de filter te triggeren (dus niet onChanged maar onSave bij druk op knop).